Assets/Script/GameManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System;
+
+public class GameManager : MonoBehaviour
+{
+    public Text PlayTime;
+    private TimeSpan startTime = new TimeSpan(0, 10, 00);
+
+    private void Awake()
+    {
+
+    }
+
+    private void Start()
+    {
+        InvokeRepeating("CountDown", 0, 1);
+    }
+
+
+
+    private void CountDown()
+    {
+        startTime = startTime.Subtract(new TimeSpan(0, 0, 1));
+        PlayTime.text = $"{startTime.Minutes.ToString()} : {startTime.Seconds.ToString()}";
```

Comment:
```suggestion
        PlayTime.text = remainingTime.ToString(@"mm\:ss")
```

User string formatting. More effective expression.

Reply:
와 이런 방법도 있었네요 ㅎㅎ